SimplenoteUITests/Generic.swift

```diff
@@ -154,8 +154,10 @@ class WebView {
 
     class func tapDone() {
         let doneButton = app.buttons[UID.Button.done]
-        guard doneButton.exists else { return }
-        doneButton.tap()
+        guard doneButton.exists else {
+            return
+        }
+        doneButton.tapCenterCoordinates(in: app)
```

Comment:
For some reason the webkit done button couldn't be tapped on by the test runner.  So getting the center point and tapping the coords instead